.github/workflows/cd-publish.yml

```
@@ -1,4 +1,4 @@
-name: CI - FAST Publish
+name: Release Packages
```

```suggestion
name: Release NPM Packages
```
If we're looking for clarity I might suggest being specific about the type of packages.